apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.scss

```diff
@@ -0,0 +1,8 @@
+@use '~@kirbydesign/core/src/scss/utils';
+
+:host {
+  display: block;
+  width: 100%;
```

Comment:
Very minor thing, but I don't believe you need this width here.

Reply:
Done